app/src/main/java/org/p2p/wallet/swap/ui/jupiter/settings/presenter/SwapCommonSettingsMapper.kt

```diff
@@ -0,0 +1,120 @@
+package org.p2p.wallet.swap.ui.jupiter.settings.presenter
+
+import androidx.annotation.StringRes
+import org.p2p.core.common.DrawableContainer
+import org.p2p.core.common.TextContainer
+import org.p2p.uikit.components.finance_block.FinanceBlockCellModel
+import org.p2p.uikit.components.finance_block.FinanceBlockStyle
+import org.p2p.uikit.components.left_side.LeftSideCellModel
+import org.p2p.uikit.components.right_side.RightSideCellModel
+import org.p2p.uikit.model.AnyCellItem
+import org.p2p.uikit.organisms.sectionheader.SectionHeaderCellModel
+import org.p2p.uikit.utils.image.ImageViewCellModel
+import org.p2p.uikit.utils.text.TextViewCellModel
+import org.p2p.wallet.R
+
+class SwapCommonSettingsMapper {
+
+    fun getNetworkFeeCell(): FinanceBlockCellModel {
+        return FinanceBlockCellModel(
+            leftSideCellModel = LeftSideCellModel.IconWithText(
+                firstLineText = TextViewCellModel.Raw(
+                    text = TextContainer(R.string.swap_settings_network_fee_title),
+                ),
+                secondLineText = TextViewCellModel.Raw(
+                    text = TextContainer(R.string.swap_settings_network_fee_subtitle),
+                    textColor = R.color.text_mint,
+                ),
+            ),
+            rightSideCellModel = RightSideCellModel.SingleTextTwoIcon(
+                text = TextViewCellModel.Raw(
+                    text = TextContainer(R.string.swap_settings_network_fee_free),
+                    textColor = R.color.text_mint,
+                ),
+                firstIcon = ImageViewCellModel(
+                    icon = DrawableContainer(R.drawable.ic_info_outline),
+                    iconTint = R.color.icons_mint,
+                )
+            ),
+            payload = SwapSettingsPayload.NETWORK_FEE,
+            styleType = FinanceBlockStyle.BASE_CELL,
+        )
+    }
+
+    fun getSlippageList(slippage: Double): List<AnyCellItem> = mutableListOf<AnyCellItem>().apply {
+        val selectedSlippage = slippage.slippageToEnum()
+        addSlippageCell(
+            slippage = TextContainer("0,1%"),
+            isSelected = selectedSlippage == SwapSlippagePayload.ZERO_POINT_ONE,
+            payload = SwapSlippagePayload.ZERO_POINT_ONE
+        )
+        addSlippageCell(
+            slippage = TextContainer("0,5%"),
+            isSelected = selectedSlippage == SwapSlippagePayload.ZERO_POINT_FIVE,
+            payload = SwapSlippagePayload.ZERO_POINT_FIVE
+        )
+        addSlippageCell(
+            slippage = TextContainer("1%"),
```

Comment:
to string

Reply:
what do you mean? SwapSlippagePayload.CUSTOM from res

Reply:
ah yeah, i just glitched, sorry